comment_scraper.py

```diff
@@ -1,87 +1,95 @@
-# This sample shows you how to scrape comments from a project.
-# Pass the root_asset_id and a developer token in to get_all_project_comments
-# to scrape the comments from a project. 
+###################################
+# This scraper shows you how to gather comments from
+# a Frame.io project and write to a tidy CSV.
+# Comments are gathered recursively from the 
+# Folders, files and version stacks within your project.
+###################################
 
-from frameioclient import FrameioClient
-import requests, json, csv, itertools
-
-ROOT_ASSET_ID = "Put your root asset ID here."
-TOKEN = "Put your developer token here."
-
-# Function for grabbing comments from assets. It's used by
-# get_all_project_comments. It takes an initialized client,
-# asset_id, and reference to an empty list. It returns
-# all comments on all assets.
-
-def all_comments(client, asset_id, comment_list):
-    files = client.get_asset_children(asset_id)
-
-    for asset in files:
-        if asset['type'] == "file":
-            if asset['comment_count'] > 0:
 
-# You can't get the asset name or the parent ID from the get_comments call, so we are saving it
-# from the get_asset_children call, so we can append it to each comment for use
-# later.
-
-                asset_parent_id = asset['parent_id']
-                asset_name = asset['name']
-                comments = client.get_comments(asset['id'])
-                my_comment_list = [comment for comment in comments.results]
-                for object in my_comment_list:
-                    object.update({'parent_id':asset_parent_id})
-                    object.update({'name':asset_name})
-                comment_list.append(my_comment_list)
-
-        if asset['type'] == "folder":
-            if asset['item_count'] > 0:
-                all_comments(client, asset['id'], comment_list)
+from frameioclient import FrameioClient
+import csv
+from os import getenv
+from itertools import chain
+
+# Retrieve token and root_asset_id from a local .env file.
+# If you don't know what root_asset_id is, read this guide: https://docs.frame.io/docs/root-asset-ids 
+token = getenv('FRAME_IO_TOKEN')
```

Comment:
May be good to check for None and give the user some feedback, in case these are not set.

Reply:
Also this may be a personal preference, but keeping these all caps actually feels more right to me. Easier to remember later that they are coming from the environment when/if that's relevant.